lms/routes.py

```diff
@@ -108,3 +108,4 @@ def includeme(config):
 
     config.add_route("lti.oidc", "/lti/1.3/oidc")
     config.add_route("lti.jwks", "/lti/1.3/jwks")
+    config.add_route("lti.deeplinking.formfields", "/lti/1.3/deeplinking/formfields")
```

Comment:
```suggestion
    config.add_route("lti.deep_linking.form_fields", "/lti/1.3/deep_linking/form_fields")
```
I don't think "formfields" is a thing either?